crt_portal/cts_forms/views.py

```diff
@@ -916,6 +916,21 @@ def _notification_change(request):
     return redirect(reverse('crt_forms:crt-forms-notifications'))
 
 
+@login_required
+def resources_view(request):
+    resources = Resource.objects.all()
+    data = []
+    for index, resource in enumerate(resources):
+        data.append({
+            'resource': resource,
+            'tags': list(map(str, resource.tags.values_list('name', flat=True))),
+        })
+    data_dict = {
+        'data_dict': data,
+    }
```

Comment:
This makes a neat comprehension:

```suggestion
    resources = [
        {
            'resource': resource,
            'tags': list(map(str, resource.tags.values_list('name', flat=True))),
        }
        for index, resource in Resource.objects.all()
    ]
    data_dict = {
        'data_dict': resources,
    }
```